depth_bg_remove:
  ros__parameters:
    depth_topic: /camera/d405_camera/depth/image_rect_raw
    camera_info_topic: /camera/d405_camera/depth/camera_info #intrinsics info
    bounding_box_topic: /viewpoint_generation/bounding_box_marker #bounding box topic
    #depth map filtering range
    dmap_filter_min: 0.07
    dmap_filter_max: 0.50

    viz_enable: true
    publish_pointcloud: true
    pcd_downsampling_stride: 1
    #frame_ids 
    target_frame: object_frame #frame of the target object
    main_camera_frame: eoat_camera_link #frame of the eoat camera
    #cropped parameters in eoat camera frame
    crop_radius: 0.05
    crop_z_min: 0.05
    crop_z_max: 0.40
    
    standoff_m: 0.10 # standoff_value for fixed standoff mode
    standoff_mode: euclidean